### PR TITLE
dependency

### PR DESCRIPTION
**Pull request by anil-kk**
requires @angular-devkit/core

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "prepublishOnly": "npm run build:lib"
   },
   "devDependencies": {
+    "@angular-devkit/core": "0.0.28",
```
Comment (jkuri) [inline package.json]: why is this needed?

Comment (anil-kk) [reply package.json]: When i tried to start the app. I got errors that this particular package is needed. When I installed this package locally everything went smooth. I could able to run the app locally on my computer. 
image

Comment (jkuri) [reply package.json]: okay, thanks for explanation. merging...
